test/withTheme_test.js

```diff
@@ -221,16 +245,33 @@ describe("withTheme", () => {
         type: "object",
         properties: { foo: { type: "string" }, bar: { type: "string" } },
       };
-      let { node } = createComponent(withTheme({ ...themeTemplates }), {
-        schema,
-        ...userTemplates,
-      });
+      let { node } = createComponent(
+        WrapperClassComponent({ ...themeTemplates }),
+        {
+          schema,
+          ...userTemplates,
+        }
+      );
       expect(
         node.querySelectorAll(".with-theme-field-template")
       ).to.have.length.of(0);
       expect(node.querySelectorAll(".user-field-template")).to.have.length.of(
         1
       );
     });
+
+    it("should forward the ref", () => {
+      const ref = createRef();
+      const schema = {};
+      const uiSchema = {};
+
+      createComponent(withTheme({}), {
+        schema,
+        uiSchema,
+        ref,
+      });
+
+      expect(ref.current.submit).to.exist();
```

Comment:
This will make the test pass:

```suggestion
      expect(ref.current.submit).not.eql(undefined);;
```